index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Microverse Project by Chris</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="stylesheet" href="assets/css/desktop.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="/">My Logo</a>
+      <button onclick="return;" type="button">
+        <span>
+          <img
+            src="assets/images/icons/Union.png" 
+            alt="nav button">
+        </span>
+      </button>
+    </nav>
+  </header>
+  <main>
+    <div class="backgrounds">
```

Comment:
- [ ] Kindly wrap the whole section with the `section` tag as It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
- Such as: 
`<main>`
`  <section>`
`    <div>`
`    </div>`
`  </section>`
`</main>`

Reply:
Done.